packages/editor/src/components/block-list/index.js

```diff
@@ -189,7 +189,6 @@ class BlockList extends Component {
 				{ map( blockClientIds, ( clientId, blockIndex ) => (
 					<BlockListBlock
 						key={ 'block-' + clientId }
-						index={ blockIndex }
```

Comment:
I spent 2 hours trying to find why the blocks were rerendering, it turns out we had a useless prop (that was not even used :P)